pkg/obfuscator/multi_obfuscator_test.go

```diff
@@ -60,7 +60,7 @@ func TestMultiObfuscationPaths(t *testing.T) {
 func TestMultiObfuscationReport(t *testing.T) {
 	mo := NewMultiObfuscator(
 		[]ReportingObfuscator{
-			&splitObfuscator{tracker: NewSimpleTracker()},
+			&splitObfuscator{tracker: NewSimpleTracker(map[string]string{})},
```

Comment:
now it's a bit too late, but you could've added another New* method instead of filling all the defaults here.

`NewSimpleTracker()` could just use an empty map underneath and `NewSimpleTrackerMap(map...)` would be a new method based off of a map. Less code to change :)

Reply:
Done